### PR TITLE
Feature: add extra relationships between users and tasks

### PR DESCRIPTION
**Pull request by AlfredoPrograma**
# What
- Add new relationships between users and tasks:
    - createdBy
    - assignee
    - lastUpdatedBy
    - deletedBy

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -46,9 +46,13 @@ model User {
...
   name               String?
   isTutorialFinished Boolean             @default(false)
-  tasks              Task[]
   workspaces         UsersOnWorkspaces[]
 
+  creatorOnTasks  Task[] @relation("TasksCreator")
+  assigneeOnTasks Task[] @relation("TasksAssignee")
+  updaterOnTasks  Task[] @relation("TasksUpdater")
+  deleterOnTasks  Task[] @relation("TasksDeleter")
+
```
Comment (juanzenn) [inline prisma/schema.prisma:55]: 1. Change names on the model

`createdTasks` Task[]
`assignedTasks` Task[]

2. Delete these properties: updaterOnTasks, deleterOnTasks

**Review by juanzenn — CHANGES_REQUESTED** [review]
**PLEASE FIX**

File: prisma/schema.prisma (hunk 1 above)
Comment (AlfredoPrograma) [reply prisma/schema.prisma:53]: Done

**Comment by AlfredoPrograma** [comment]
Omg this is another PR made by @AlfredoPrograma. Woow! Amazing database desing.

Are u agree @juanzenn @pedrouzcategui ?

**Review by juanzenn — APPROVED** [review]

**Comment by juanzenn** [comment]
Brilliant. I like your PR @AlfredoPrograma. We'll fix so much in the future. Stay tuned.

**Pull request merged**